### DISCUSSION TITLE
Webhook

### DISCUSSION THREAD

**Discussion by wbankowski** [allegro/allegro-api#6710] [Suggestions - rejected]
Czy jest dostępne w Allegro API lub planowane do uruchomienia wysyłanie webhooks ze zdarzeniami dotyczącymi transakcji (zamówień, zmian statusów, płatności, itp.)? Przydałoby się takie narzędzie pod potrzeby implementacji automatyzacji z narzędziami typu Airtable, Make, Zappier.

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię - na tę chwilę nie planujemy wprowadzać takiej funkcjonalności.
